contracts/market/CreateLoanWithNFTFacet.sol

```diff
@@ -146,6 +163,26 @@ contract CreateLoanWithNFTFacet is ReentryMods, PausableMods {
                 );
                 allowedLoanSize_ += baseLoanSize * amounts[i];
             }
+        } else if (version == 3) {
```

Comment:
remove this `else if` from here. we will add the additional checks we need in the `MainnetCreateLoanWithNFTFacet`